khd/kernels/scattermoe/triton_implementation/kernels.py

```diff
@@ -26,9 +26,9 @@ def scatter2scatter_triton_kernel(
     block_start_idx_ptr,
     FAN_OUT,
     M,
```

Comment:
M should be also constexpr here?
to make autotune work with this?

Reply:
icic good catch. actually `M` changes alot so im not sure if we should remove it from `autotune`

Reply:
yes agree